wolvic/BUILD.gn

```diff
@@ -334,6 +334,7 @@ android_library("wolvic_java") {
     "//components/embedder_support/android:web_contents_delegate_java",
     "//components/url_formatter/android:url_formatter_java",
     "//content/public/android:content_full_java",
+    "//content/public/android:content_main_dex_java",
```

Comment:
I am not getting any error. How do I trigger it?

Reply:
`gn check out/{dir}` but not sure if it triggered only by the debug args.